test/modules/aws-account-integration.ts

```diff
@@ -121,13 +129,38 @@ describe('AwsAccount Integration Testing', () => {
     })();
   });
 
-  it('inserts a second, useless row into the aws_account table', query(`
-    INSERT INTO aws_account (access_key_id, secret_access_key, region)
-    VALUES ('fake', 'creds', 'us-west-2')
+  it('inserts a second, useless row into the aws_credentials table', query(`
+    INSERT INTO aws_credentials (access_key_id, secret_access_key) VALUES ('fake', 'creds')
   `));
 
   it('does absolutely nothing when you apply this', apply());
 
+  it('selects a default region', query(`
+    UPDATE aws_regions SET is_default = TRUE WHERE region = '${process.env.AWS_REGION}';
+  `));
+
+  it('confirms that the default region was set', query(`
+    SELECT * FROM aws_regions WHERE is_default = TRUE;
+  `, (res: any[]) => expect(res.length).toBe(1)));
+
+  it('tries to set a second default region', query(`
+    UPDATE aws_regions SET is_default = TRUE WHERE region = 'us_east_1';
+  `));
+
+  it('confirms that the default region was not changed', query(`
+    SELECT * FROM aws_regions WHERE is_default = TRUE;
+  `, (res: any[]) => {
+    expect(res.length).toBe(1);
+    expect(res[0].region).toBe(process.env.AWS_REGION);
+  }));
+
+  it('updates the default region with the handy `default_aws_region` function', query(`
+    SELECT * FROM default_aws_region('us_east_1');
```

Comment:
nit: maybe it makes sense to use another region rather than the fallback value to ensure it is working?

Reply:
also, the regions are not supposed to be `us-east-1` instead of `us_east_1`?